gcloud/operation.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrap long-running operations returned from Google Cloud APIs."""
+
+from google.longrunning import operations_pb2
+
+
+_GOOGLE_APIS_PREFIX = 'types.googleapis.com'
+
+_TYPE_URL_MAP = {
+}
```

Comment:
it must've gotten dropped when I was rebasing.  Both `gcloud.bigtable.instance` and `gcloud.bigtable.cluster` should be calling `_register_type_url` when imported.